vdev/src/app.rs

```diff
@@ -43,6 +44,10 @@ pub fn set_repo_dir() -> Result<()> {
     env::set_current_dir(path()).context("Could not change directory")
 }
 
+pub fn version() -> Result<String> {
+    env::var("VERSION").or_else(|_| util::read_version())
```

Comment:
Please call this in the `version` command to avoid them getting out of sync.

Reply:
I will implement this now